deploy-cloudrun/README.md

```diff
@@ -0,0 +1,178 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# deploy-cloudrun
+
+This action deploys your container image to [Cloud Run][cloud-run] and makes the URL
+available to later build steps via outputs.
+
+## Prerequisites
+
+This action requires Google Cloud credentials that are authorized to deploy a
+Cloud Run service. See the Authorization section below for more information.
+
+## Usage
+
+```yaml
+steps:
+- id: deploy
+  uses: GoogleCloudPlatform/github-actions/deploy-appengine@master
+  with:
+    image: gcr.io/cloudrun/hello
+    service: hello-cloud-run
+    credentials: ${{ secrets.gcp_credentials }}
+
+# Example of using the output
+- id: test
+  run: curl "${{ steps.deploy.outputs.url }}"
+```
+
+## Inputs
+
+- `image`: Name of the container image to deploy (e.g. gcr.io/cloudrun/hello:latest).
+  Required if not using a service YAML.
+
+- `service`: ID of the service or fully qualified identifier for the service.
+  Required if not using a service YAML.
+
+- `region`: Region in which the resource can be found.
+
+- `credentials`: Service account key to use for authentication. This should be
+  the JSON formatted private key which can be exported from the Cloud Console. The
+  value can be raw or base64-encoded. Required if not using a the
+  `setup-gcloud` action with exported credentials.
+
+- `env_vars`: List of key-value pairs to set as environment variables in the format:
+  KEY1=VALUE1,KEY2=VALUE2. All existing environment variables will be
+  removed first.
+
+- `metadata`: YAML serivce description for the Cloud Run service. See
+  [Metadata customizations](#metadata-customizations) for more information.
+
+- `project_id`: (Optional) ID of the Google Cloud project. If provided, this
+  will override the project configured by gcloud.
+
+### Metadata customizations
+
+You can store your service specification in a YAML file. This will allow for
+further service configuration, such as [memory limits](https://cloud.google.com/run/docs/configuring/memory-limits),
+[CPU allocation](https://cloud.google.com/run/docs/configuring/cpu),
+[max instances](https://cloud.google.com/run/docs/configuring/max-instances),
+and [more.](https://cloud.google.com/sdk/gcloud/reference/run/deploy#OPTIONAL-FLAGS)
+
+- See [Deploying a new service](https://cloud.google.com/run/docs/deploying#yaml)
+to create a new YAML service definition, for example:
+
+```YAML
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: SERVICE
+spec:
+  template:
+    spec:
+      containers:
+      - image: IMAGE
+```
+
+- See [Deploy a new revision of an existing service](https://cloud.google.com/run/docs/deploying#yaml_1)
+to generated a YAML service specification from an existing service:
+
+```
+gcloud run services describe SERVICE --format yaml > service.yaml
+```
+## Allow unauthenticated requests
+
+A Cloud Run product recommendation is that CI/CD systems not set the
```

Comment:
I think this wording does not make sense with respect to the run action as `--allow-unauthenticated` is a gcloud flag and we are not exposing any option for this in the action.

Reply:
updated.